docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -173,25 +199,21 @@ The Azure CLI has the following commands to manage role assignments:
 * [az role assignment create](/cli/azure/role/assignment#az-role-assignment-create)
 * [az role assignment delete](/cli/azure/role/assignment#az-role-assignment-delete)
 
-We recommend you use the **Contributor** role at minimum for a service principal. This role has full permissions to read and write to an Azure account. The **Reader** role is more restrictive, with read-only access. For more information on Role-Based Access Control (RBAC) and roles, see [RBAC: Built-in roles](/azure/active-directory/role-based-access-built-in-roles).
+The **Contributor** role has full permissions to read and write to an Azure account. The **Reader** role is more restrictive, with read-only access. For more information on Role-Based Access Control (RBAC) and roles, see [RBAC: Built-in roles](/azure/active-directory/role-based-access-built-in-roles).
 
-This example adds the **Reader** role and removes the **Contributor** one:
+This example adds the **Reader** role and removes the **Contributor** role:
 
 ```azurecli-interactive
-az role assignment create --assignee APP_ID --role Reader
-az role assignment delete --assignee APP_ID --role Contributor
+az role assignment create --assignee appID --role Reader
```

Comment:
This command will very likely to change in the future since we will not default `--scope` to subscription just like what we did for `az ad sp create-for-rbac`. Better to specify `--scope` as well.

Reply:
thanks for the suggestion, @jiasli .  Done.